index.html

```diff
@@ -71,7 +72,220 @@ <h1>I’m Sylvester,
+        <i class="fas fa-circle fa-xs"></i>
+        <span class="head-item">2015</span>
+      </div>
+      <div class="card-para">
+        <p>
+          A daily selection of privately personalized reads; no accounts
+          or sign-ups required.
+        </p>
+      </div>
+      <div class="card-lang">
+        <ul>
+          <li class="html">html</li>
+          <li class="css">css</li>
+          <li class="js">javaScript</li>
+        </ul>
+      </div>
+      <div class="seeport">
+        <span>
+          <a href="#">See Project</a>
+        </span>
+      </div>
+    </section>
+    <section class="about">
+      <div class="about-me">
+        <h1>About <br /> Myself</h1>
+      </div>
+      <div class="about-text">
+        <p>
+          Hello I’m a software developer! I can help you build a product, 
+          feature or website Look through some of my work and experience! 
+          If you like what you see and have a project you need coded, 
+          don’t hestiate to contact me.
+        </p>        
+      </div>
+      <a href="#" class="connect2">LET'S CONNECT</a>
+      <div class="social-icons2">
+        <ul>
+          <li>
+            <a href="#">
+              <img src="images/twitter.png" alt="Connect on Twitter">
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="images/I.png" alt="Connect on Twitter">
+              <span>
+                <img src="images/n.png" alt="">
+              </span>
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="images/connect-m.png" alt="Connect on IM">
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="images/github.png" alt="Connect on GitHub">
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="images/thumbs-up.png" alt="Two fingers">
+            </a>
+          </li>
+        </ul>
+      </div>
+      <div class="getresume">
+        <span>
+          <a href="#">Get my resume</a>
+        </span>
+      </div>
+    </section>
+    <section class="language">
+      <div class="language-head">
+        <span>Languages</span>
+        <span><i class="fas fa-chevron-down"></i></span>
+      </div>
+      <div class="java">
+        <img src="images/javascript.png" alt="Javascript" />
+        <span>JavaScript</span>
+      </div>
+      <div class="java">
+        <img src="/portfolio-project/images/html.png" alt="HTML" />
+        <span>HTML</span>
+      </div>
+      <div class="java">
+        <img src="images/css.png" alt="CSS" />
+        <span>CSS</span>
+      </div>
+    </section>
+    <section class="extra-skill">
+      <div class="frame">
+        <span>Frameworks</span>
+        <i class="fas fa-chevron-right"></i>
+      </div>
+    <hr />
+      <div class="skill">
+        <span>Skills</span>
+        <i class="fas fa-chevron-right"></i>
+      </div>
```

Comment:
- [ ] The `frame` and `skill` elements should be on the same level as the `language element. So, please remove the extra-skill `section` tag. As I have stated above, these elements are better defined in `li` tag.

Reply:
Thanks for these feedbacks, implementing them made the code better.

Please, I cant seem to achieve a horizontal line inside the `<ul>` element. My local linters keep warning that `<hr >` can't be a child to `<ul>`. Please, any suggestions on how this can be achieved?

Reply:
- [ ] This issue is not well addressed.  As I already mentioned, you should have made `frame` and `skill` elements on the same level as the `language` element, on their own. Such a semantic approach will help browsers, developers, and crawlers to distinguish between different types of data. 
- [ ] For the horizontal line, you can use `border-bottom` styling for the elements instead of `<hr>`.